easybuild/easyconfigs/r/RFdiffusion/RFdiffusion-1.1.0-foss-2022a.eb

```diff
@@ -1,56 +1,156 @@
-RFdiffusion is an open source method for structure generation, with or without conditional information 
-(a motif, target etc). It can perform a whole range of protein design challenges as we have outlined 
-in the RFdiffusion paper.
-"""
+description = """RFdiffusion is an open source method for structure generation, with or without  
+conditional information (a motif, target etc). It can perform a whole range of  
+protein design challenges as we have outlined in the RFdiffusion paper          
+https://www.biorxiv.org/content/10.1101/2022.12.09.519842v1
+
+Model files are available in $RFDIFFUSION_MODELS"""
 
 toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {}
+
+_models_url = 'http://files.ipd.uw.edu/pub/RFdiffusion'
+_pysp = '%(installdir)s/lib/python%(pyshortver)s/site-packages'
+_models_dst = '%(installdir)s/models/'
+_schedules_dst = '%(installdir)s/schedules/'
+
+_models = [
+    ("Base_ckpt.pt", "6f5902ac237024bdd0c176cb93063dc4"),
+    ("Complex_base_ckpt.pt", "e29311f6f1bf1af907f9ef9f44b8328b"),
+    ("Complex_Fold_base_ckpt.pt", "60f09a193fb5e5ccdc4980417708dbab"),
+    ("InpaintSeq_ckpt.pt", "74f51cfb8b440f50d70878e05361d8f0"),
+    ("InpaintSeq_Fold_ckpt.pt", "76d00716416567174cdb7ca96e208296"),
+    ("ActiveSite_ckpt.pt", "5532d2e1f3a4738decd58b19d633b3c3"),
+    ("Base_epoch8_ckpt.pt", "12fc204edeae5b57713c5ad7dcb97d39"),
+    ("Complex_beta_ckpt.pt", "f572d396fae9206628714fb2ce00f72e"),
+    ("RF_structure_prediction_weights.pt", "1befcb9b28e2f778f53d47f18b7597fa"),
+]
+
+checksums = [
+    # schedules_v1.1.0.zip
+    '9fbb47ddb92f1c0cf234bec39ba6ba64d179501fbff53a4d10b4e55afd75e44a',
+    # Base_ckpt.pt
+    '0fcf7d7c32b4848030aca3a051e6768de194616f96ba6c38186351a33bfc6eca',
+    # Complex_base_ckpt.pt
+    '76e4e260aefee3b582bd76b77ab95d2592e64f00c51bf344968ab9239f3250bc',
+    # Complex_Fold_base_ckpt.pt
+    '0ac3b4024aea811078cec41482528291d6d7d7084bf8190ec118f54642fb81a1',
+    # InpaintSeq_ckpt.pt
+    '3b71b2b954e87d46b75a88ba64e0420fbf27f592604b10b6c3561b8c8ab70ab6',
+    # InpaintSeq_Fold_ckpt.pt
+    '51849c9fe64c16a38fe41c75db76abe044e4d3493926f6cfd29a5bde0331b7cc',
+    # ActiveSite_ckpt.pt
+    'beca1f672049161df0bc6a2d2523828f19fd9c8a2b449988e246dde42e7ea986',
+    # Base_epoch8_ckpt.pt
+    'b8e5d57f0b8a8f8cb30779c106b75210b46a914a4d19fb180676ae647f5ae23d',
+    # Complex_beta_ckpt.pt
+    '5a0b1cafc23c60b1aabcec1e49391986ac4fd02cc1b6b4cc41714ca9fe882e9e',
+    # RF_structure_prediction_weights.pt
+    '6414e9e60b0b01011e5a182def40b4e6de4e137554c887b2916d43566733ed95',
+]
+
+
+components = [('schedules', version, {
+    'easyblock': 'CmdCp',
+    'source_urls': ['https://files.ipd.uw.edu/krypton/'],
+    'sources': [{
+        'filename': '%(name)s_v%(version)s.zip',
+        'download_filename': '%(name)s.zip',
+        'extract_cmd': 'unzip -j %s'
+    }],
+    'cmds_map': [('.*', '')],
+    'files_to_copy': [(['*.pkl'], _schedules_dst)],
+})]
+
+components += [
+    (_lname, _lver, {
```

Comment:
I see some downsides to this approach of installing the model datasets of RFdifussion:
* all model files are unversioned, so I suspect that these datasets will probably be updated live without any filename change, breaking the checksums in the process.
* by adding the model datasets as `sources` they will get stored at least twice, one copy in `sourcepath` and another one more for each installation directory. Since the total size of these files is ~4 GB, that is not negligible.

I suggest to install the models in the `postinstall` step, with a simple `wget` command without checksumming. Something in the lines to what we do in [NLTK-3.7-foss-2021b.eb](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/NLTK/NLTK-3.7-foss-2021b.eb), [Kraken2-2.1.2-gompi-2022a.eb](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/k/Kraken2/Kraken2-2.1.2-gompi-2022a.eb) or [antiSMASH-5.2.0-foss-2020b.eb](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/a/antiSMASH/antiSMASH-5.2.0-foss-2020b.eb)

Reply:
@lexming I moved the model download to postinstall.